plugins/huaweicloud/src/main/java/org/eclipse/xpanse/plugins/huaweicloud/manage/HuaweiCloudVmStateManager.java

```diff
@@ -142,7 +142,7 @@ private EcsClient getEcsClient(ServiceStateManageRequest serviceStateManageReque
 
     private boolean checkEcsExecResultByJobId(EcsClient ecsClient, String jobId) {
         ShowJobResponse response = ecsClient.showJobInvoker(new ShowJobRequest().withJobId(jobId))
-                .retryTimes(huaweiCloudRetryStrategy.getRetryMaxAttempts())
+                .retryTimes(huaweiCloudRetryStrategy.getSdkRetryMaxAttempts())
```

Comment:
hi @baixinsui I think we are still mixing two things. 
In the SDK there are two types of retry 
1. http.retry - which we do when the http request fails. 
2. batch job status - retry  - here we only check for the status of the job by polling the response status. 

We must separate these two retries.

Reply:
I set a fixed max value to the retry times for waiting job success.